index.js

```diff
@@ -1,17 +1,54 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item); //adicionar um novo item
 
-  get(pos) {}
+    //esse array deve estar organizado em ordem crescente
+    this.items.sort((a, b) => a - b);
 
-  max() {}
+    this.length = this.items.length;
+    //manter o length atualizado
+  }
 
-  min() {}
+  get(pos) {
+    //garante que seja ordem crescente porque está chamando o add(item) que faz essa coisa
+    if (this.items[pos]) return this.items[pos];
+    throw new Error("OutOfBounds");
```

Comment:
Aqui vc pode retornar um ternário:

return this.items[pos] ? this.items[pos] : throw new Error('OutOfBounds');

Fica legal assim :-)